ipn/ipnlocal/local.go

```diff
@@ -450,6 +452,10 @@ type LocalBackend struct {
 	// Each callback is called exactly once in unspecified order and without b.mu held.
 	// Returned errors are logged but otherwise ignored and do not affect the shutdown process.
 	shutdownCbs set.HandleSet[func() error]
+
+	// auditLogger manages audit logging for the backend.  It queues, persists, and sends audit logs
```

Comment:
// auditLogger, if non-nil, managed audit logging for the backend.
//
// It queues, persists, ...